tests/db/test_bigchain_api.py

```diff
@@ -216,12 +88,7 @@ def test_text_search(self, b, alice):
                                  asset=asset3).sign([alice.private_key])
 
         # create the block
```

Comment:
this needs to be removed or changed

Reply:
Are you referring to the comment about creating the block, or something in the code?

Reply:
the comment, we don't create a block here anymore